indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,130 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelecteion() {
+        $('table.i-table input.select-row').on('change', function() {
+            $(this).closest('tr').toggleClass('selected', this.checked);
+            $('.js-requires-selected-row').toggleClass('disabled', !$('.sessions input:checkbox:checked').length)
+        }).trigger('change');
+    }
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    global.setupSessionsList = function setupSessionsList() {
+        handleRowSelecteion();
+        setupTableSorter();
+
+        $('#select-all').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', true).trigger('change');
+        });
+
+        $('#select-none').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', false).trigger('change');
+        });
+
+        $('.sessions .show-session-slots').on('click', function() {
+            $(this).parent().parent().next('tbody.session-blocks').toggle();
+        });
+
+        $('.sessions .toolbar').on('click', '.disabled', function(e) {
+            e.preventDefault();
+            e.stopPropagation();
+        });
+
+        $('#add-new-session').on('click', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        window.location.reload();
+                    }
+                }
+            });
+        });
+
+        $('.edit-session').on('click', function() {
+            var $this = $(this);
+            ajaxDialog({
+                url: $this.data('href'),
+                title: $this.data('title'),
+                onClose: function(data) {
+                    if (data) {
+                        window.location.reload();
+                    }
+                }
+            });
+        });
+
+        $('.show-sessions-btn').on('click', function() {
+            $(this).closest('tr').nextUntil('tr:not(.session-blocks-row)', 'tr').toggle();
+        });
+
+        $('#remove-selected-sessions').on('indico:confirmed', function(evt) {
+            evt.preventDefault();
+            var $this = $(this);
+            var sessionIds = $('.sessions input:checkbox:checked').map(function() {
+                return $(this).val();
+            }).get();
+
+            $.ajax({
+                url: $this.data('href'),
+                method: 'POST',
```

Comment:
Isn't it an overkill if we are sure that a specific request will use only one HTTP method?